core/src/lightclient/mod.rs

```diff
@@ -82,4 +122,61 @@ impl StarknetClient {
         let block = starknet_block_to_eth_block(starknet_block);
         Ok(block)
     }
+
+    /// Get the number of transactions in a block given a block id.
+    /// The number of transactions in a block.
+    ///
+    /// # Arguments
+    ///
+    ///
+    ///
+    /// # Returns
+    ///
+    ///  * `block_number(u64)` - The block number.
+    ///
+    /// `Ok(Bytes)` if the operation was successful.
+    /// `Err(LightClientError)` if the operation failed.
+    async fn get_code(
+        &self,
+        ethereum_address: Address,
+        starknet_block_id: StarknetBlockId,
+    ) -> Result<Bytes, LightClientError> {
+        // Convert the Ethereum address to a hex-encoded string
+        let address_hex = hex::encode(ethereum_address);
+        // Convert the hex-encoded string to a FieldElement
+        let ethereum_address_field_element = FieldElement::from_hex_be(&address_hex).unwrap();
+
+        // Prepare the calldata for the get_starknet_contract_address function call
+        let tx_calldata_vec = vec![ethereum_address_field_element];
+        let request = FunctionCall {
+            contract_address: self.kakarot_account_registry,
+            entry_point_selector: selector!("get_starknet_contract_address"),
```

Comment:
let's create a `selectors.rs` in this folder with only `pub const` declarations of selectors that we may need (we'll add them on one by one as we go) so that we're "type-safe" and centralise all selectors
wdyt

Reply:
Agree, this will def make it a lot safer!